ae-ease-to-gsap-customease.jsx

```diff
@@ -108,87 +108,75 @@
 
 	var curItem = app.project.activeItem;
 	var framerate = curItem.frameRate;
-	var selectedLayers = curItem.selectedLayers;
 	var selectedProperties = curItem.selectedProperties;
 
-	if (selectedLayers.length === 0) {
-		alert('Please Select at least one Layer');
-		return;
-	}
-
 	if (selectedProperties.length === 0) {
 		alert('Please Select at least one Property (Scale, Opacity, etc)');
 		return;
 	}
 
-	for (var i = 0; i < selectedLayers.length; i++) {
-		for (var f in selectedProperties) {
-			if (!{}.hasOwnProperty.call(selectedProperties, f)) {
-				continue;
-			}
+	for (var f = 0; f < selectedProperties.length; f++) {
```

Comment:
Very good catch! These few lines were left over from the copy-pasted code that was the base of this script, I totally didn't notice the problems here.